plugin.video.arteplussept/resources/settings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <setting
+        id="lang"
+        type="enum"
+        label="30050"
+        values="fr|de|en|es|pl|it"
+        default="0"/>
+    <!-- <setting id="prefer_vost"      type="bool"   label="30051" default="false"/> -->
+    <setting
+        id="quality"
+        type="enum"
+        label="30052"
+        values="SQ (High)|EQ (Medium)|HQ (Low)"
```

Comment:
Isn’t that the other way around? HQ means High Quality and SQ means Standard Quality.

Reply:
This is the way Arte labels the video qualities. I agree this looks weird, but I prefer to keep it this way just in case some users are accustomed to the Atre labels.